diff -urN MIM/as-itrans.mim ../m17n-db-1.4.0-Modified/MIM/as-itrans.mim
--- MIM/as-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/as-itrans.mim	2007-07-18 15:19:23.000000000 +0530
@@ -142,6 +142,11 @@
   ((KP_8) "৮")
   ((KP_9) "৯")
   ((KP_0) "০")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")
   ((C-@) "‍"))
 
diff -urN MIM/gu-itrans.mim ../m17n-db-1.4.0-Modified/MIM/gu-itrans.mim
--- MIM/gu-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/gu-itrans.mim	2007-07-18 15:20:50.000000000 +0530
@@ -143,6 +143,11 @@
   ((KP_8) "૮")
   ((KP_9) "૯")
   ((KP_0) "૦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")				; not in ITRANS Gujarati table
   ((C-@) "‍"))				; not in ITRANS Gujarati table
 
diff -urN MIM/hi-itrans.mim ../m17n-db-1.4.0-Modified/MIM/hi-itrans.mim
--- MIM/hi-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/hi-itrans.mim	2007-07-18 15:22:18.000000000 +0530
@@ -164,6 +164,11 @@
   ((KP_8) "८")
   ((KP_9) "९")
   ((KP_0) "०")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")			      ; not in ITRANS Devanagari table
   ((C-@) "‍"))			      ; not in ITRANS Devanagari table
 
diff -urN MIM/kn-itrans.mim ../m17n-db-1.4.0-Modified/MIM/kn-itrans.mim
--- MIM/kn-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/kn-itrans.mim	2007-07-18 15:22:47.000000000 +0530
@@ -156,6 +156,11 @@
   ((KP_8) "೮")
   ((KP_9) "೯")
   ((KP_0) "೦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-") 
   ((S-\ ) "‌")				; not in ITRANS Kannada table
   ((C-@) "‍"))				; not in ITRANS Kannada table
 
diff -urN MIM/ml-itrans.mim ../m17n-db-1.4.0-Modified/MIM/ml-itrans.mim
--- MIM/ml-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/ml-itrans.mim	2007-07-18 15:23:12.000000000 +0530
@@ -139,6 +139,11 @@
   ((KP_8) "൮")
   ((KP_9) "൯")
   ((KP_0) "൦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")
   ((C-@) "‍"))
 
diff -urN MIM/or-itrans.mim ../m17n-db-1.4.0-Modified/MIM/or-itrans.mim
--- MIM/or-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/or-itrans.mim	2007-07-18 15:23:37.000000000 +0530
@@ -143,6 +143,11 @@
   ((KP_8) "୮")
   ((KP_9) "୯")
   ((KP_0) "୦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")
   ((C-@) "‍"))
 
diff -urN MIM/pa-itrans.mim ../m17n-db-1.4.0-Modified/MIM/pa-itrans.mim
--- MIM/pa-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/pa-itrans.mim	2007-07-18 15:23:59.000000000 +0530
@@ -136,6 +136,11 @@
   ((KP_8) "੮")
   ((KP_9) "੯")
   ((KP_0) "੦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")				; not in ITRANS Gurmukhi table
   ((C-@) "‍"))				; not in ITRANS Gurmukhi table
 
diff -urN MIM/te-itrans.mim ../m17n-db-1.4.0-Modified/MIM/te-itrans.mim
--- MIM/te-itrans.mim	2007-05-29 14:44:23.000000000 +0530
+++ ../m17n-db-1.4.0-Modified/MIM/te-itrans.mim	2007-07-18 15:24:57.000000000 +0530
@@ -146,6 +146,11 @@
   ((KP_8) "౮")
   ((KP_9) "౯")
   ((KP_0) "౦")
+  ((KP_Decimal) ".")
+  ((KP_Divide) "/")
+  ((KP_Multiply) "*")
+  ((KP_Add) "+")
+  ((KP_Subtract) "-")
   ((S-\ ) "‌")				; not in ITRANS Telugu table
   ((C-@) "‍"))				; not in ITRANS Telugu table
 
